scripts/import/import_variant_synonyms

```diff
@@ -147,6 +197,356 @@ sub download_PharmGKB{
   return "rsid.tsv";
 }
 
+=head2 download_UniProt
+
+collect current export from UniProt site
+
+=cut 
+
+sub download_UniProt{
+
+  my $http     = HTTP::Tiny->new();
+  my $response = $http->get('https://www.uniprot.org/docs/humsavar.txt');
+  die "Failed to pick up file\n" unless $response->{success};
+
+  open my $out, ">humsavar.txt" ||die "Failed to write data locally : $!\n";
+  print $out  $response->{content}; 
+
+  return "humsavar.txt";
+}
+
+=head2 parse_uniprot
+
+parse UniProt file 
+
+=cut 
+
+sub parse_uniprot {
+  my $infile = shift;
+  
+  my %synonym;
+  my @phenotypes;
+  
+  # Open the input file for reading
+  if($infile =~ /gz$/) {
+    open (IN, "zcat $infile |") or die ("Could not open $infile for reading");
+  }
+  else {
+    open(IN,'<',$infile) or die ("Could not open $infile for reading");
+  }
+  
+  # Read through the file and parse out the desired fields
+  while (<IN>) {
+    chomp;
+    
+    # A regexp to catch the meta information in the header. Just echo this to the stdout for logging purposes
+    if ($_ =~ m/^\s*(Description|Name|Release)\:/) {
+      print STDOUT $_ . "\n";
+    }
+    
+    # Main regexp to extract relevant variation information
+    if ($_ =~ m/^(\S+)\s+\S+\s+(VAR\_\d+)\s+\w\.\S+\s+(Disease|Polymorphism|Unclassified)\s+(\-|rs\d*)\s*(.*)$/) {
+      
+      # Get the data that was caught by the regexp
+      my $gene = $1;
+      my $uniprot_id = $2;
+      my $rs_id = $4;
+      my $phenotype = $5;
+      
+      # If no rsId was given, will attempt to get one by looking up the Uniprot id in the synonym table
+      if ($rs_id ne '-') {
+        push(@{$synonym{$rs_id}},$uniprot_id);
+      }
+      #else {
```

Comment:
Maybe you could removed these commented lines as we don't import phenotype data for this source anymore

Reply:
Thanks! I deleted it.